release-pipeline.yml

```diff
@@ -110,6 +110,10 @@ stages:
               GOARCH=amd64 GOOS=linux go build -tags "netgo,se_integration" -o "$(root_dir)/azcopy_linux_se_amd64"
             displayName: 'Generate Linux AMD64 SE Integration'
 
+          - script: |
```

Comment:
Could you please run the pipeline on this branch once to verify if the changes are working as expected?

Reply:
Done. https://dev.azure.com/azstorage/AzCopy-NextGen/_build/results?buildId=23352&view=results